resources/sass/pages/_home.scss

```diff
@@ -0,0 +1,104 @@
+#categories-id {
+    .home-section {
+        padding: 33px 0px;
+        max-width: 1140px;
+        margin: 0 auto;
+        position: relative;
+        .row {
+            @include div-row;
+            text-align: center;
+            flex-direction: row;
+        }
+        &:nth-child(odd) {
+            .row {
+                flex-direction: row-reverse;
+            }
+        }
+        .offer {
+            display: none;
+            width: 100%;
+        }
+        .w3-transparent {
+            background-color: #7f7f7f !important;
+        }
+        .w3-left,
+        .w3-right {
+            cursor: pointer
+        }
+        .w3-badge {
+            height: 7px;
+            width: 7px;
+            padding: 3px;
+            margin: 1%;
+            cursor: pointer
+        }
+        .w3-grey {
+            background-color: #999999 !important;
+            opacity: 0.6;
+            font-weight: 400;
+        }
+        .w3-button {
+            color: $color_white !important;
+            &:hover {
+                color: $color_white !important;
+            }
+        }
+        .w3-content {
+            max-width: 1150px !important;
+        }
+        .cat_button {
+            background: $color_red;
+            color: $color_white;
+            border-color: $color_red;
+            padding: 1%;
+            font-weight: 500;
+            font-size: 1em;
+            /* line-height: 2%; */
+        }
+        .cat-inner-heading {
+            font-family: $primary_font !important;
+            font-weight: 700 !important;
+            font-size: 22px !important;
+        }
+        .cat-inner-img {
+            width: 60%;
+        }
+        .cat-inner_text {
+            font-weight: 500;
+            font-size: 15px;
+            /* line-height: 50px; */
+            padding: 2%;
+        }
+    }
+    .home-section:after {
+        content: '';
+        display: block;
+        position: absolute;
+        bottom: -1rem;
+        background: radial-gradient(farthest-corner at 50% 0px, grey 0%, transparent 50%);
+        width: 100%;
+        height: 1rem;
+    }
+    /* for mobile */
+    @media screen and (max-width: 480px) {
+        /* for ipad  menu css */
+        .cat-inner-img {
+            width: 125% !important;
+            margin-top: 31%;
+        }
+        .w3-btn,
+        .w3-button {
+            border: none;
+            display: inline-block;
+            padding: 2px 13px;
+        }
+    }
+}
+
+@media screen and (max-width: 767px) {
+    .cat-inner-img {
+        width: 116% !important;
```

Comment:
OMG, @shilpaRai123 do you even know how widths work? Why your widths are reaching more than 100% ?